website/docs/reference/resource-configs/trino-configs.md

```diff
@@ -0,0 +1,335 @@
+Note that some connectors in Trino have limited or no support for `MERGE`.
+
+```jinja2
+{{
+    config(
+      materialized = 'incremental',
+      unique_key='user_id',
+      incremental_strategy='merge',
+      )
+}}
+select * from {{ ref('users') }}
+{% if is_incremental() %}
+  where updated_ts > (select max(updated_ts) from {{ this }})
+{% endif %}
+```
+
+#### Incremental overwrite on hive models
+
+In case that the target incremental model is being accessed with
+[hive](https://trino.io/docs/current/connector/hive.html) Trino connector, an `insert overwrite`
+functionality can be achieved when using:
+
+```ini
+<hive-catalog-name>.insert-existing-partitions-behavior=OVERWRITE
+```
+
+setting on the Trino hive connector configuration.
+
+Below is a sample hive profile entry to deal with `OVERWRITE` functionality for the hive connector called `minio`:
+
+```yaml
+trino-incremental-hive:
+  target: dev
+  outputs:
+    dev:
+      type: trino
+      method: none
+      user: admin
+      password:
+      catalog: minio
+      schema: tiny
+      host: localhost
+      port: 8080
+      http_scheme: http
+      session_properties:
+        minio.insert_existing_partitions_behavior: OVERWRITE
+      threads: 1
+```
+
+Existing partitions in the target model that match the staged data will be overwritten.
+The rest of the partitions will be simply appended to the target model.
+
+NOTE that this functionality works on incremental models that use partitioning:
+
+```jinja2
+{{
+    config(
+        materialized = 'incremental',
+        properties={
+          "format": "'PARQUET'",
+          "partitioned_by": "ARRAY['day']",
+        }
+    )
+}}
+```
+
+### Materialized view
+
+The adapter also supports [materialized views](https://trino.io/docs/current/sql/create-materialized-view.html).
+At every subsequent `dbt run` command, the materialized view is [refreshed](https://trino.io/docs/current/sql/refresh-materialized-view.html).
+
+You can also define custom properties for the materialized view through the `properties` config.
+
+This materialization supports the [full_refresh](https://docs.getdbt.com/reference/resource-configs/full_refresh) config and flag.
+Whenever you want to rebuild your materialized view, e.g. when changing underlying SQL query, run `dbt run --full-refresh`.
+
+In model add:
+
+```jinja2
+{{
+  config(
+    materialized = 'materialized_view',
+    properties = {
+      'format': "'PARQUET'"
+    },
+  )
+}}
+```
+
+or in `dbt_project.yaml`:
+
+```yaml
+models:
+  path:
+    materialized: materialized_view
+    properties:
+      format: "'PARQUET'"
+```
+
+### Snapshots
+
+Commonly, analysts need to "look back in time" at some previous state of data in their mutable tables. While some source data systems are built in a way that makes accessing historical data possible, this is often not the case. dbt provides a mechanism, snapshots, which records changes to a mutable table over time.
```

Comment:
I think we could just link to the snapshot docs. I think the note about the timestamps should be kept but we don't need to explain snapshots in this way when it's already on our docs